frontend/src/components/Footer/styles.js

```diff
@@ -0,0 +1,14 @@
+import { yellow } from '@material-ui/core/colors';
+import { makeStyles } from '@material-ui/core/styles';
+import { sizing } from '@material-ui/system';
+
+const useStyles = makeStyles((theme) => ({
+    title: {
+        backgroundColor: theme.palette.primary.main,
+        color: theme.palette.primary.contrastText,
+        padding: '10px 0',
+        width: '20%'
+    },
+}));
```

Comment:
Jeśli to ma stylować footer to daj width: `'100%' `.
Wrapper w którym będzie znajdować się będzie miał jakieś wymiary i stopka uplasuje się na dole ekranu.
W takiej sytuacji dając mu 100% (rodzica) będzie pasował idealnie   

![image](https://user-images.githubusercontent.com/41823689/112888000-1c779e80-90d4-11eb-963a-cbf15050d9b6.png)

Reply:
Ustawiłam to tak, żeby tymczasowo jako tako wyglądało. Jest już utworzony task na zrobienie HOC ze stopką i headerem oraz {childem} pomiędzy. Ten children to będzie cała aktualna zawartość apki, która się zmienia, a cały to będzie tym kontenerkiem narzucającym rozmiar